From 36ecfe9327f9c65b30fbe02434117dbd50a5d0d3 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Thu, 12 Dec 2024 01:27:08 +0800
Subject: [PATCH 298/316] AOSCOS: MIPS: loongson3-memset: replace
 <asm/export.h> with <linux/export.h>

Per commit 9259e15b3f27 ("mips: replace #include <asm/export.h> with
for <linux/export.h>.

Follow the commit and revise loongson3-memset.S accordingly.

Fixes: "AOSCOS: memset optimization for loongson-3"
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 arch/mips/loongson64/loongson3-memset.S | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/mips/loongson64/loongson3-memset.S b/arch/mips/loongson64/loongson3-memset.S
index 227b2d67082e..0a35c013f6df 100644
--- a/arch/mips/loongson64/loongson3-memset.S
+++ b/arch/mips/loongson64/loongson3-memset.S
@@ -8,9 +8,9 @@
  * Copyright (C) 2007 by Maciej W. Rozycki
  * Copyright (C) 2011, 2012 MIPS Technologies, Inc.
  */
+#include <linux/export.h>
 #include <asm/asm.h>
 #include <asm/asm-offsets.h>
-#include <asm/export.h>
 #include <asm/regdef.h>
 
 #define LONG_S_L sdl
-- 
2.48.1

